### ISSUE TITLE
Expose `physics::weld(ent1, ent2)`

### ISSUE THREAD

**Issue by philpax** [AmbientRun/Ambient#161]
This should allow building a larger object from multiple smaller ones.

The existing `weld_two` can panic - we should ensure that's fixed before exposing it to scripting.

**Comment by ten3roberts** [comment]
Agreed. The existing weld will also cause the physx fetchResults issue as it manually calls it, rather than waiting for synchronization.
